consoles/VNC.pm

```diff
@@ -1093,13 +1030,11 @@ sub _receive_colour_map ($self) {
     $self->socket->read(my $map_infos, 5);
     my ($padding, $first_colour, $number_of_colours) = unpack('Cnn', $map_infos);
 
-    for (my $i = 0; $i < $number_of_colours; $i++) {
+    for (0 .. $number_of_colours) {
         $self->socket->read(my $colour, 6);
         my ($red, $green, $blue) = unpack('nnn', $colour);
-        tinycv::set_colour($self->vncinfo, $first_colour + $i, $red / 256, $green / 256, $blue / 256);
+        tinycv::set_colour($self->vncinfo, $first_colour + $_, $red / 256, $green / 256, $blue / 256);
```

Comment:
this includes `$number_of_colours`. before it was `$i < $number_of_colours`.

Reply:
right, fixed. thx